client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/cloud-project-compute-infrastructure-virtualMachine-addEdit.html

```diff
@@ -1218,6 +1218,44 @@ <h4 data-translate="cpcivm_addedit_advanced_options_description"></h4>
                                                 </span>
                                             </td>
                                         </tr>
+                                        <!-- available but not activated regions -->
+                                        <tr data-ng-repeat="region in VmAddEditCtrl.availableRegions.data"
+                                            style="cursor: not-allowed;"
+                                            data-ng-if="!VmAddEditCtrl.availableRegions.loading && VmAddEditCtrl.availableRegions.data.length > 0">
+                                            <td style="width: 50px;">
```

Comment:
Please don't use `style`, can you find another way to do this ? Add the `w-50` class for example

Reply:
https://getbootstrap.com/docs/4.1/utilities/sizing/

Reply:
bootstrap only supports width in %

Reply:
customised bootstrap and added support for 10, 15 and 20% w-10, w-15, w-20